examples/dao/scripts/run/clear_vote_record.js

```diff
@@ -0,0 +1,31 @@
+const { executeTx } = require('./common/common.js');
+const { types } = require('@algo-builder/web');
+const { accounts } = require('./common/accounts.js');
+
+async function clearVoteRecord (deployer, voterAcc, proposalLsig) {
```

Comment:
how about renaming `proposalLsig` to `proposalAddr`? In other places, only in scripts, as well (but only when we use it as an address, not the TEAL).

Reply:
> but only when we use it as an address

yes, let's rename to proposalAddr in these cases.